Simplenote/src/androidTest/java/com/automattic/simplenote/BaseUITest.kt

```diff
@@ -0,0 +1,43 @@
+package com.automattic.simplenote
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import com.automattic.simplenote.models.Tag
+import com.automattic.simplenote.utils.TagUtils
+import com.automattic.simplenote.utils.TestBucket
+import org.junit.Before
+import java.security.SecureRandom
+
+open class BaseUITest {
```

Comment:
Minor (🔍): Consider reformatting this class. I believe the formatting for this class can be further improved (see `charPool`).

Reply:
Thanks, I have reformat the file.